plugins/alignments/src/AlignmentsFeatureDetail/AlignmentsFeatureDetail.tsx

```diff
@@ -73,36 +64,17 @@ interface AlnInputProps {
   model: any // eslint-disable-line @typescript-eslint/no-explicit-any
 }
 
-function Formatter({ value }: { value: unknown }) {
-  const [show, setShow] = useState(false)
-  const display = String(value)
-  if (display.length > 100) {
-    return (
-      <>
-        <button type="button" onClick={() => copy(display)}>
-          Copy
-        </button>
-        <button type="button" onClick={() => setShow(val => !val)}>
-          {show ? 'Show less' : 'Show more'}
-        </button>
-        <div>{show ? display : `${display.slice(0, 100)}...`}</div>
-      </>
-    )
-  }
-  return <div>{display}</div>
-}
-
 const AlignmentFeatureDetails: FunctionComponent<AlnInputProps> = props => {
-  const { model } = props
+  const { model, omit = [] } = props
   const feat = JSON.parse(JSON.stringify(model.featureData))
   return (
     <Paper data-testid="alignment-side-drawer">
-      <BaseFeatureDetails
+      <BaseFeatureDetails {...props} />
```

Comment:
I was not seeing the performance hit from viewing a long read that was seen previously, so this was removed